src/main/java/neqsim/physicalproperties/PhysicalPropertyType.java

```diff
@@ -0,0 +1,34 @@
+package neqsim.physicalproperties;
+
+import neqsim.util.exception.InvalidInputException;
+
+/**
+ * Types of PhysicalProperties, e.g. density, viscosity, conductivity.
+ *
+ * @author ASMF
+ */
+
+public enum PhysicalPropertyType {
+  DENSITY, VISCOSITY, CONDUCTIVITY;
```

Comment:
DENSITY ➔ MASS_DENSITY
Reflects that it’s the density related to mass in a fluid context.

VISCOSITY ➔ DYNAMIC_VISCOSITY
Highlights that it’s the fluid’s resistance to flow.

CONDUCTIVITY ➔ THERMAL_CONDUCTIIVITY
Clarifies that it’s related to heat transfer rather than electrical conductivity.